### ISSUE TITLE
Licencing conflict

### ISSUE THREAD

**Issue by Admer456** [bacontsu/halflife-extended-master#1]
`LICENSE.md` contains the CC-BY 4.0 licence, but Half-Life SDK is licenced under the proprietary HL SDK EULA, which prevents any sublicencing (look at `source-code/LICENSE`). 

To remain compliant with HL SDK EULA, you should **change the licence** from CC-BY 4.0 to the SDK's actual licence, or explicitly limit the CC-BY licence to files that are not covered by the EULA. (your own `README.md` and potentially any headers and sources that don't include any HL SDK headers or anything that is protected by the EULA)

If you want to get credited, which I believe is the reason why you went with CC-BY in the first place, there isn't much you can legally do. However, you can look into alternatives, e.g. simply asking people to credit you, something like "Please credit [XYZ] if you use this in your project".

TL;DR do something about your licence pls, thank you :3

**Comment by bacontsu** [comment]
alrighty, the license has been reverted back to the normal HLSDK Eula on the new commit. thank you for your help!
https://github.com/bacontsu/halflife-extended-master/commit/89cc72698f778c85b674f64efc0a03c87483c345
